pages/blog/conference-2023.md

```diff
@@ -0,0 +1,158 @@
+They provided valuable industry insights, and we are proud of the network we built during the events.
+
+<Figure
+  src="/img/posts/2023-summary/speaker-2023.webp"
+/>
+
+
+You can also catch up on the live recording from [AACoT'23 Madrid](https://www.youtube.com/playlist?list=PLbi1gRlP7pihD-7XYb6c9wcTdkavDzF3_) and [AACoT'23 Bangalore](https://www.youtube.com/playlist?list=PLbi1gRlP7pijVocLZS7FeWKA4NBzJa7_Z).
+
+## Special Shout-Out
+
+I want to express my gratitude to these individuals and many others for their unwavering dedication in assisting with AACoT'23 behind the scenes. 
+They contributed to the design of website prototypes, conference design implementation, creating promotional videos, reviewing Call for Papers (CfPs), editing conference recordings, and providing constant communication and support. I am genuinely thankful for their hard work and commitment.
+
+<Profile profiles={[
+{
+    name: 'Aishat Muibudeen',
+    avatar: 'https://avatars.githubusercontent.com/Mayaleeeee',
+    link: 'https://github.com/Mayaleeeee'
+  }, 
+  {
+    name: 'Lukasz Gornicki',
+    avatar: 'https://avatars.githubusercontent.com/derberg',
+    link: 'https://github.com/derberg'
+  },
+  {
+    name: 'Azeez Elegbede',
+    avatar: 'https://avatars.githubusercontent.com/acethecreator',
+    link: 'https://github.com/acethecreator'
+  },
+  {
+    name: 'Barbanio González',
+    avatar: 'https://avatars.githubusercontent.com/Barbanio',
+    link: 'https://github.com/Barbanio'
+  },
+  {
+    name: 'Iván García Sainz-Aja',
+    avatar: 'https://avatars.githubusercontent.com/ivangsa',
+    link: 'https://github.com/ivangsa'
+  },
+  {
+    name: 'Dale Lane',
+    avatar: 'https://avatars.githubusercontent.com/dalelane',
+    link: 'https://github.com/dalelane'
+  },
+  {
+    name: 'Souvik De',
+    avatar: 'https://avatars.githubusercontent.com/Souvikns',
+    link: 'https://github.com/Souvikns'
+  },
+  {
+    name: 'Karuna Tata',
+    avatar: 'https://avatars.githubusercontent.com/starlightknown',
+    link: 'https://github.com/starlightknown'
+  }, 
+    {
+    name: 'Jonas Lagoni',
+    avatar: 'https://avatars.githubusercontent.com/jonaslagoni',
+    link: 'https://github.com/jonaslagoni'
+  },
+  {
+    name: 'Florence Njeri',
+    avatar: 'https://avatars.githubusercontent.com/Florence-Njeri',
+    link: 'https://github.com/Florence-Njeri'
+  }
+]} />
+
+## Where To Next?
+
+As we plan for AACoT'24, we have three potential locations - one in Lagos (happening early Q2), U.K. hosted by APIdays London, and the other in France at APIdays Paris(happening in Q4). If you are interested, we are currently seeking sponsors for AACoT'24, either as our financial sponsor or event and host sponsor.
+
+You can check out our [Open Collective Page](https://opencollective.com/asyncapi/events/asyncapi-conference-on-tour-6b3c0aa1) to see the current sponsorship tiers or download our sponsorship prospectus for more information.
```

Comment:
great idea with renaming conference project from last year - this way we have already a bunch of people there that we can update and make aware about onferences in 2024